pytorch_lightning/loops/dataloader/evaluation_loop.py

```diff
@@ -166,7 +176,12 @@ def _reload_evaluation_dataloaders(self) -> None:
         elif self.trainer.val_dataloaders is None or self.trainer._should_reload_dl_epoch:
             self.trainer.reset_val_dataloader()
 
-    def _on_evaluation_start(self, *args: Any, **kwargs: Any) -> None:
+        if not self.trainer.sanity_checking and self._dataloader_state_dict:
+            # ready didn't happen yet, so we can't use `current_dataloader_idx`.
```

Comment:
it's still using it in the line below ^^

Reply:
Not sure to understand.

Reply:
the comment contradicts the line below. it is using current_dataloader_idx

Reply:
the comment contradicts the line below. it is using current_dataloader_idx

Reply:
Good catch !